nf_core/pipeline-template/.github/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# {{ name }}: Contributing Guidelines
+# `{{ name }}`: Contributing Guidelines
```

Comment:
I'm not sure if by adding the '`' the `{{name}}` will still be changed.

Reply:
That should be fine, it won't cause any problems.